.gitignore

```diff
@@ -20,3 +23,8 @@ Thumbs.db
 # Secure files #
 ################
 hummingbird/hummingbird/settings_secret.py
+
+# Databases #
+####################
+db.sqlite3
+
```

Comment:
nit: Unnecessary blank line at the bottom. It's good to end the file with a newline, but this file has two newlines at the end.